src/main.cpp

```diff
@@ -158,7 +168,6 @@ int main()
 
     wait(2);    // safety feature: be able to re-flash before starting
     control_timer_start(CONTROL_FREQUENCY);
-    wait(0.1);  // necessary to prevent MCU from randomly getting stuck here if PV panel is connected before battery
```

Comment:
This wait call is still necessary for the MPPT-1210-HUS and should be kept.

Reply:
Interesting, never noticed that I removed this. Will add it back.